src/lvl_script_commands.c

```diff
@@ -1066,16 +1065,19 @@ static void special_transfer_creature_check(const struct ScriptLine* scline)
 
 static void script_transfer_creature_check(const struct ScriptLine* scline)
 {
+    ALLOCATE_SCRIPT_VALUE(scline->command, scline->np[0])
```

Comment:
command_add_value at the bottom of the function also does the ALLOCATE, should be replaced with a PROCESS_SCRIPT_VALUE(scline->command);

Reply:
should be good now